plugins/carla/carla-bridge.hpp

```diff
@@ -0,0 +1,198 @@
+/*
+ * Carla plugin for OBS
+ * Copyright (C) 2023 Filipe Coelho <falktx@falktx.com>
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#pragma once
+
+#include "CarlaBackend.h"
+#include "CarlaBridgeUtils.hpp"
+
+#include <QtCore/QByteArray>
+#include <QtCore/QProcess>
+#include <QtCore/QString>
+
+#include <vector>
+
+// generates warning if defined as anything else
+#define CARLA_API
+
+// import macro from OBS, purposefully not including any other OBS APIs here
+#define MAX_AV_PLANES 8
```

Comment:
That's not much of an "import" - it requires remembering that `carla-bridge` makes use of this value and if it's updated in `libobs` that it also needs to be updated here, which is a sure-fire point of failure.

Either `carla-bridge` can be made to work without this value, or `libobs` needs to gain an API function that can be used to query the value from it.